index.js

```diff
@@ -34,12 +36,14 @@ export {
 	Icon,
 	Image,
 	Input,
+	isServerSide,
 	Link,
 	MediaItem,
 	Overline,
 	Paragraph,
 	Pill,
 	Separator,
 	Stack,
+	useInterval,
```

Comment:
Should we namespace utility functions or leave them intermixed with the components?

Reply:
in the past, we've intermixed them

Reply:
I don't really have an issue with doing so. makes exports easier

Reply:
if we do, it would have to be another story, going to resolve this conversation.